server/jobs/helpers.js

```diff
@@ -27,3 +36,27 @@ export const getMailInfo = async (placement) => {
     startDate: dateStart === '' ? 'not specified' : dateStart,
   };
 };
+
+/**
+ * @function sendPlacementFetchEmail
+ * @desc Send email to ESA if fetching placements fails constantly
+ * @param {string} receiver Info about the mail to be sent
+ *
+ * @returns {Object} Promise to fetch new placements and execute automations
+ */
+
+const sendPlacementFetchEmail = async (receiver) => {
```

Comment:
Nice implementation. I think the `async` and `await` are not necessary since there is nothing below the awaited method call